src/network/netdev/fou-tunnel.h

```diff
@@ -0,0 +1,45 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#pragma once
+
+/***
+  This file is part of systemd.
+
+  Copyright 2018 Susant Sahani
+***/
+
+#if HAVE_FOU_CMD_GET
```

Comment:
Where these `HAVE_FOU_*` are defined? I guess that meson.build update is missing in this commit.